data/crates.json

```diff
@@ -748,20 +748,36 @@
                     ]
                 },
                 {
-                    "slug": "grpc",
-                    "name": "gRPC",
+                    "slug": "rpc",
+                    "name": "RPC",
```

Comment:
This diff still feels suboptimal to me. It defines a generic RPC category but then sort of fills it with a bunch of gRPC/protobuf-related crates that are usually all used together. If you really want to specifically call out protobuf as an encoding mechanism, I think it would be better to change the existing Serialization category and maybe specifically discuss self-describing vs not-self-describing and also text vs binary serde encodings. This is kind of a topic by itself -- I think we should probably mention rkyv and postcard on the site as well.

Reply:
> I think it would be better to change the existing Serialization category and maybe specifically discuss self-describing vs not-self-describing and also text vs binary serde encoding

This feels better to me, I agree that it's kind of shoehorning things into RPC currently. Is that what the group wants?

Reply:
Pushed some changes to try to address this:
- Reverted gRPC to previous form
- Hoisted `Serialization` to a subgroup-level section in `Common`. 
- Added an intro description on serialization differentiating between the main types (text vs binary, self-describing vs non-self-describing with and withou schema)
- Added purposes for `Non-self-describing, external schema file` and `Non-self-describing, no external schema file`
- Added `prost`, `capnp`, `flatbuffers`, `postcard`, `rykv`

It's a bit verbose now. There isn't really a great place to tuck additional general context, so the subsection description wound up long.

It also is moving outside of pure `what is the blessed rust crate` into `what serialization format should i use, and what crate supports that`. I think that's kind of the nature of the beast here though since the two questions go hand in hand.

What I have now, I think is a decent 'at a glance' description of the tradeoffs between the various widely used crates. My hope is that a reader knowing they need to deserialize, could pick something 'off the shelf' with this without needing to dive into blog posts doing comparisons.

But, please push back if this is going too off the rails and no longer meeting the goals of this page. Very glad to adjust further / prune it down again.